example-playbooks/replace_node/replace_node.yml

```diff
@@ -198,3 +198,38 @@
   hosts: scylla-monitor
   roles:
     - ansible-scylla-monitoring
+
```

Comment:
In the patch description: s/Manually repairing/Repairing/
"Manual" means that actual person has to do that.

Reply:
Done